### PR TITLE
#1141 Added virtual_disks to nb_inventory.py

### PR DESCRIPTION
**Pull request by tkn2023**
Added virtual_disks support for inventory.

## Related Issue

#1141 

## New Behavior

This adds virtual_disks to the inventory file. Multiple disks are also supported.

## Contrast to Current Behavior

Due to the new feature of virtual disks this was not included in the inventory.

## Discussion: Benefits and Drawbacks

- With this addition you are able to use virtual_disks in an automatic script

## Changes to the Documentation

virtual_disks is now added to nb_inventory and can be used with "virtual_disks: True"

## Proposed Release Note Entry

Virtual_disks has been added to nb_inventory.py and can be used with "virtual_disks: True"

## Double Check

Please put an x into the brackets (like `[x]`) if you've completed that task.

* [X] I have read the comments and followed the [CONTRIBUTING.md](https://github.com/netbox-community/ansible_modules/blob/devel/CONTRIBUTING.md).
* [X] I have explained my PR according to the information in the comments or in a linked issue.
* [X] My PR targets the `devel` branch.

### CODE REVIEW

**Comment by sc68cal** [comment]
This needs to add to the `integration` tests where we have some devices in the JSON file that have virtual disks, to exercise this code.

**Review by sc68cal — CHANGES_REQUESTED** [review]
needs integration tests

**Comment by tkn2023** [comment]
> This needs to add to the `integration` tests where we have some devices in the JSON file that have virtual disks, to exercise this code.

I have not found a JSON file that have virtual disks in them?

**Comment by sc68cal** [comment]
You need to add to existing JSON files or create new ones

File: tests/integration/targets/inventory-v3.7/files/test-inventory-bearer-token.json [hunk 1]
```diff
@@ -44,27 +44,28 @@
...
-                            "display": "Test Nexus One",
-                            "id": 4,
-                            "name": "Test Nexus One"
+                            "display": "Test Nexus Child One",
+                            "id": 5,
+                            "name": "Test Nexus Child One",
+                            "url": "http://localhost:32768/api/dcim/devices/5/"
                         },
-                        "display": "Ethernet1/1",
+                        "display": "Ethernet2/1",
```
Comment (sc68cal) [inline tests/integration/targets/inventory-v3.7/files/test-inventory-bearer-token.json:52]: Sorry, why were these changed? Your commit message does not indicate the reason

Comment (sc68cal) [reply tests/integration/targets/inventory-v3.7/files/test-inventory-bearer-token.json:52]: The reason I ask is because it appears that you changed existing test files instead of creating new ones in the inventory that tests your specific feature.

**Review by sc68cal — DISMISSED** [review]
Made changes to existing devices in inventory, instead of creating _additional_ devices in the inventory test files

File: tests/integration/targets/inventory-v3.7/files/test-inventory-bearer-token.json (hunk 1 above)
Comment (tkn2023) [reply tests/integration/targets/inventory-v3.7/files/test-inventory-bearer-token.json:52]: The inventory files were created with the .hacking.
I have just adjusted these files according to the output that was generated with it. I added virtual disks to the netbox_deploy file and have not changed any existing devices.
I used a clean install of netbox with netbox-docker and done everything according to Contribution.

Comment (sc68cal) [reply tests/integration/targets/inventory-v3.7/files/test-inventory-bearer-token.json:52]: Ok thank you for the explanation
